hack/ci-e2e.sh

```diff
@@ -55,6 +55,13 @@ export PATH="/usr/local/go/bin:${PATH}"
 
 # Build the container image with e2e tag (used in tests)
 IMG=quay.io/metal3-io/baremetal-operator:e2e make docker
+# Pull the ironic image and extract tftpboot files
+docker pull quay.io/metal3-io/ironic:release-26.0
+docker create --name tmp-ironic quay.io/metal3-io/ironic:release-26.0
```

Comment:
the ironic release looks a bit outdated, is that done on purpose?

Reply:
Oh I missed it while rebasing. Still not quite sure about this part to be honest. Virtualmedia seems to be working at least but I will need to work on this a bit more I think